<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyDialog</class>
 <widget class="QDialog" name="MyDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>436</width>
    <height>76</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Box</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="4">
    <widget class="QLineEdit" name="EditBox">
     <property name="toolTip">
      <string>Point format: CODE, DESCRIPTION
Pattern format: *</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QRadioButton" name="PointRadio">
     <property name="text">
      <string>Point</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QRadioButton" name="PatternRadio">
     <property name="text">
      <string>Pattern</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="AcceptButton">
     <property name="text">
      <string>Accept</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="RejectButton">
     <property name="text">
      <string>Reject</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>AcceptButton</sender>
   <signal>clicked()</signal>
   <receiver>MyDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>258</x>
     <y>80</y>
    </hint>
    <hint type="destinationlabel">
     <x>279</x>
     <y>74</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>RejectButton</sender>
   <signal>clicked()</signal>
   <receiver>MyDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>377</x>
     <y>79</y>
    </hint>
    <hint type="destinationlabel">
     <x>407</x>
     <y>104</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
